scipy/optimize/_optimize.py

```diff
@@ -1015,6 +1020,7 @@ def check_grad(func, grad, x0, *args, epsilon=_epsilon,
     ...     return x[0]**2 - 0.5 * x[1]**3
     >>> def grad(x):
     ...     return [2 * x[0], -1.5 * x[1]**2]
+    >>> import numpy as np
```

Comment:
This should move to the top of the Example section.

Reply:
I'll fix this now.